src/client/components/pages/entities/author.js

```diff
@@ -40,7 +40,7 @@ const {deletedEntityMessage, extractAttribute, getTypeAttribute, getEntityUrl,
 	ENTITY_TYPE_ICONS, getSortNameOfDefaultAlias, transformISODateForDisplay} = entityHelper;
 const {Button, Col, Row} = bootstrap;
 
-function AuthorAttributes({author, averageRating}) {
+function AuthorAttributes({author, averageRating, reviewsCount}) {
```

Comment:
A question about this: If I followed correctly, the `reviews` object should already be in the `author` object; why are we passing the ratings props separately?

Reply:
Correct. The ratings data is inside the `author` object. I've made the changes required.